src/bptl/camunda/tests/test_execute_task.py

```diff
@@ -35,23 +35,36 @@ def setUpTestData(cls):
         config.rest_api_path = "engine-rest/"
         config.save()
 
-        Service.objects.create(
-            api_root=ZRC_URL, api_type="zrc", label="zrc",
-        )
-        Service.objects.create(
-            api_root=ZTC_URL, api_type="ztc", label="ztc_local",
-        )
-        TaskMapping.objects.create(
+        mapping = TaskMapping.objects.create(
             topic_name="zaak-initialize",
             callback="bptl.work_units.zgw.tasks.CreateZaakTask",
         )
+        DefaultServiceFactory.create(
+            task_mapping=mapping,
+            service__api_root=ZRC_URL,
+            service__api_type="zrc",
+            alias="ZRC",
+        )
+        DefaultServiceFactory.create(
+            task_mapping=mapping,
+            service__api_root=ZTC_URL,
+            service__api_type="ztc",
+            alias="ZTC",
+        )
 
     def test_execute_one(self, m):
         task = ExternalTaskFactory.create(
             topic_name="zaak-initialize",
             variables={
                 "zaaktype": {"value": ZAAKTYPE},
                 "organisatieRSIN": {"value": "123456788"},
+                "services": {
+                    "type": "Object",
```

Comment:
I think this should be
```suggestion
                    "type": "json",
```